### ISSUE TITLE
Create Community Stats Page

### ISSUE THREAD

**Issue by fzyang1227** [Code-4-Community/speak-for-the-trees-frontend#244]
Blocked by #238

**Why**
- To show community statistics surrounding trees
- Collective benefits of tree adoption/stewardship

**What to do**
- Implement Stats Page as dictated by #238 
- Allow NONE+ users to see:
    - How many trees were adopted, in total and per neighborhood, in the past specified timeframe (default ever and 1 week)
    - How many stewardship activities were performed in the past specified timeframe (default ever and 1 week) and the kinds of activities performed
- Allow ADMIN+ users to:
    - See the name and contact information of tree adopters adjacent to adoption and stewardship information
    - See the proportion of users who use mobile devices relative to desktops
    - Download community stats information as described (Reports)

**Comment by huang0h** [comment]
Closing as the page is no longer needed
